packages/react-components/react-tooltip/src/components/Tooltip/useTooltip.tsx

```diff
@@ -67,7 +67,7 @@ export const useTooltip_unstable = (props: TooltipProps): TooltipState => {
     hideDelay,
     relationship,
     visible,
-    shouldRenderTooltip: visible,
+    shouldRenderTooltip: visible && !isServerSideRender,
```

Comment:
This can be overridden to true further on in this function, which probably isn't the behavior you want in SSR. Perhaps this SSR check should be done after that? Or perhaps add the SSR check in renderTooltip instead: `shouldRenderTooltip && !isServerSideRender` or something.

Reply:
That's a good point, I've pushed https://github.com/microsoft/fluentui/pull/23612/commits/facdb928f1170b865ad6ee0b72dd57cc65810e2f which consolidates the SSR check to override `shouldRenderTooltip`  in one place

Reply:
Oops I actually missed that it was already checking for `isServerSideRender` below when I made that comment 🙈. But your latest change is cleaner regardless. Thanks!